modules/ROOT/pages/customsidebar.adoc

```diff
@@ -78,6 +78,9 @@ tinymce.init({
       icon: 'comment',
       onSetup: (api) => {
         console.log('Render panel', api.element());
+			return () => {
+		 	api.element().focus();
+			};
```

Comment:
```suggestion
        console.log('Render panel', api.element());
        return () => api.element().focus();
```
Code examples should use spaces instead of tabs as such the indentation wasn't great here.

On that note though... why are we focusing the sidebar element when the sidebar is destroy? That doesn't make any sense.

Reply:
I’ve committed @MitchCTiny’s suggested edits. They fix the indent character errors and change the sample to correct the focus error.